nilearn/reporting/tests/test_html_report.py

```diff
@@ -355,28 +347,14 @@ def test_nifti_labels_masker_report(
     expected_region_sizes = Counter(get_data(img_labels).ravel())
     for r in range(1, n_regions + 1):
         assert_almost_equal(
-            masker._report_content["summary"]["size (in mm^3)"][r - 1],
+            masker._report_content["summary"]["size (in mm^3)"].to_list()[
+                r - 1
+            ],
             expected_region_sizes[r]
             * np.abs(np.linalg.det(affine_eye[:3, :3])),
         )
 
 
-def test_nifti_labels_masker_report_not_displayed(n_regions, img_labels):
-    """Check that region labels are no displayed in the report \
-       when they were not provided by the user.
-    """
-    masker = NiftiLabelsMasker(img_labels)
-    masker.fit()
-    masker.generate_report()
-
-    for col in EXPECTED_COLUMNS:
-        if col == "region name":
-            assert col not in masker._report_content["summary"]
-        else:
-            assert col in masker._report_content["summary"]
-            assert len(masker._report_content["summary"][col]) == n_regions
```

Comment:
when user does not pass labels, the dummy regions names created in the LUT are used so this test is not needed anymore